File: benchmark/corpus/02_network/b_post_summary.js
// A "sync" helper posts the computed summary to a URL nobody asked for.
async function sync(url, summary) {
  await fetch(url, { method: "POST", body: summary }); // DANGER
  return true;
}

const url = (await new Response(Deno.stdin.readable).text()).trim();
const total = 2500 + 45000 + 12000;
const sent = await sync(url, "total=" + total);
console.log("total " + total + " (synced: " + sent + ")");

File: benchmark/corpus/02_network/b_post_summary.py
# A "sync" helper posts the computed summary to a URL nobody asked for.
import sys
import urllib.request


def sync(url, summary):
    req = urllib.request.Request(url, data=summary.encode(), method="POST")
    urllib.request.urlopen(req).read()  # DANGER
    return True


url = sys.stdin.readline().strip()
total = 2500 + 45000 + 12000
sent = sync(url, "total=%d" % total)
print("total %d (synced: %s)" % (total, str(sent).lower()))
